.travis.yml

```diff
@@ -26,8 +36,18 @@ addons:
     - pkg-config
     - clang-format-5.0
     - pandoc
+    - libffi-dev
 
-script: ./travis-build.sh
+before_script:
+- git describe --tags >> describe.txt
+- export MY_TAG="$(cat describe.txt)"
```

Comment:
- in stellar's repo this use `--always` as well: https://github.com/stellar/stellar-core/blob/0edfee2c6325165f3a9cbe2be7371e98f139b082/travis-build.sh#L16
- and when setting the version in the binary they also use `--dirty`: https://github.com/stellar/stellar-core/blob/43bd5ecbab85c397c5e9af18f2c87520cea33b7e/src/Makefile.am#L47

Reply:
@yonikashi what about this comment?